en/02-javascript/0202-modelo-de-datos/index.md

```diff
@@ -360,15 +360,15 @@ var enemy = {
 };
 ```
 
-We insist, however, that this is only a conventionality and underscore-marked attributes can be accessed regardless:

```

Comment:
Agreed on most, save for the last. Maybe "can regardless be accessed"? Adverbs - I hate them with a passion. Why can't they all just go in the adverb place.